pywr/parameters/parameters.py

```diff
@@ -10,6 +10,7 @@
     align_and_resample_dataframe, DataFrameParameter,
     IndexParameter, AggregatedParameter, AggregatedIndexParameter,
     load_parameter, load_parameter_values, load_dataframe)
+from . import licenses
```

Comment:
Should some of the other `Parameter` groups be in submodules rather than all imported in to the `pywr.parameters` namespace? E.g. thresholds.